arm-notificationhubs/2017-04-01/swagger/notificationhubs.json

```diff
@@ -0,0 +1,1956 @@
+            "type": "string",
+            "description": "The namespace name."
+          },
+          {
+            "name": "notificationHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The notification hub name."
+          },
+          {
+            "name": "authorizationRuleName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The connection string of the NotificationHub for the specified authorizationRule."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PolicykeyResource"
+            },
+            "description": "Parameters supplied to regenerate the NotificationHub Authorization Rule Key."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Regenerate the Primary/Secondary Key and returns the Keys and the connection string",
+            "schema": {
+              "$ref": "#/definitions/ResourceListKeys"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NotificationHubs/namespaces/{namespaceName}/notificationHubs/{notificationHubName}/pnsCredentials": {
+      "post": {
+        "tags": [
+          "NotificationHubs"
+        ],
+        "operationId": "NotificationHubs_GetPnsCredentials",
+        "description": "Lists the PNS Credentials associated with a notification hub .",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "namespaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace name."
+          },
+          {
+            "name": "notificationHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The notification hub name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Gets all the NotificationHub PNS credentials",
+            "schema": {
+              "$ref": "#/definitions/PnsCredentialsResource"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CheckAvailabilityParameters": {
```

Comment:
We have a separate work item to track this. Already changed the service code. We will make the necessary changes soon